man/avocado.rst

```diff
@@ -488,7 +488,7 @@ This command will run the `sleeptest.py` test, as found on the standard
 test directories. The output should be similar to::
 
     JOB ID    : <id>
-    JOB LOG   : /home/<user>/avocado/job-results/job-<date>-<shortid>/job.log
+    JOB LOG   : /home/user/avocado/job-results/job-<date>-<shortid>/job.log
```

Comment:
Hi @clebergnu great job, LGTM. I have a suggestion to use `<user>` instead of `user`. In my opinion is more clear that is a generic value, but that is my personal feeling, it's up to you.

Reply:
I feel like `/home/user/` will require less attention of readers.  People may stop and wonder, "what's the `<>`?  Anyway, we can always revisit this later.